packages/std/extra/libtool_sanitize_dependencies.bri

```diff
@@ -20,7 +20,7 @@ export function libtoolSanitizeDependencies(
   recipe: std.RecipeLike<std.Directory>,
 ): std.Recipe<std.Directory> {
   return runBash`
-    sed -i -e "/^dependency_libs='/,/'/ s#//lib/lib\\([^\\s]*\\)\\.la#-l\\1#g" "$BRIOCHE_OUTPUT"/lib/*.la
+    sed -i -e "/^dependency_libs='/,/'/ s|//lib/lib\\([^[:space:]]*\\)\\.la|-l\\1|g" "$BRIOCHE_OUTPUT"/lib/*.la
```

Comment:
I also replaced `#` by `|`, just for consistency with what we already have. This change has absolutely no impact here, it's just a character delimiter for sed rule system, and this character can be whatever we want, in fact.